### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MoonscriB**

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+|---|------------------------------------------------------------------------|------------|-----|-----|-----|-----|-----|-----|
+| 8 | jrvs-remote-desktop-centos7.us-east1-c.c.phrasal-bonus-340017.internal | 2          |  x86_64| AMD EPYC 7B12| 2249.998| 512|  8005740 | 2022-02-11 21:55:16|
+| 5 | paglio-HP-Z400-Workstation  | 4  | x86_64| Intel(R) Xeon(R) CPU |2080.817  |  256 | 16373892    | 2022-02-15 12:12:06    |
+
+* `host_usage`
+
+|   timestamp  | host_id | memory_free    |  cpu_idle   |  cpu_kernel   |  disk_io   |disk_available     |
+|-----|---------|-----|-----|-----|-----|-----|
+|  2022-02-11 22:02:35   | 1       |  3710052   |  95   |  0   |   118  |  5382048   |
+|  2022-02-12 03:12:42   | 1       |   4167032  |   96  |   0  |   75  |  5369296   |
```
Comment (trevor-jrvs) [inline linux_sql/README.md]: include generic table structure with constraints

File: linux_sql/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+* 
+* This is an instance storing the data on the local system. To be complete, it still needs to be made into a client-
+<<<<<<< HEAD
+server architecture, which would on the short term simplify the client-side program, as a side effect;
+* Installation and set-up should be automated;
+* Queries should be embedded inside easy to enter commands.
+=======
+server type of app, which would on the short term simplify the client-side program;
+* Installation and set-up should be automated.
+>>>>>>> 0ceb33554099ebb01cc180f9a699c825dd5fff20
```
Comment (trevor-jrvs) [inline linux_sql/README.md]: typos: clean up

File: linux_sql/scripts/host_info.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+if [[ ! "$psql_port" =~ ^[0-9]+$ || $psql_port -ge 65535 || $psql_port -le 0 ]]; then
+  echo "Port number must be a valid natural number no higher than 65535"
+  exit 1
+fi
+
+#save hostname to a variable and check if hostname's data as already been entered
+hostname=$(hostname -f)
+export PGPASSWORD=$psql_password
+host_id_query="SELECT id FROM host_info WHERE hostname = '$hostname';"
```
Comment (trevor-jrvs) [inline linux_sql/scripts/host_info.sh:23]: is hostname unique?

File: linux_sql/scripts/psql_docker.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+docker container inspect jrvs-psql
+container_status=$?
+
+case $cmd in
+  create)
+
+    if [ $container_status -eq 0 ]; then
+		  echo 'Container already exists'
+		  exit 1
+	  fi
```
Comment (trevor-jrvs) [inline linux_sql/scripts/psql_docker.sh:19]: indent

File: linux_sql/scripts/psql_docker.sh [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+#Here, we get to know if the container we are planning on using or creating already exists.
+docker container inspect jrvs-psql
+container_status=$?
+
+case $cmd in
+  create)
+
+    if [ $container_status -eq 0 ]; then
+		  echo 'Container already exists'
+		  exit 1
```
Comment (trevor-jrvs) [inline linux_sql/scripts/psql_docker.sh]: exit 0

File: linux_sql/scripts/psql_docker.sh [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+      echo "Container has not been created."
+      exit 1
+    fi
+
+    #Starting or stopping, depending on the "cmd" argument
+	  docker container $cmd jrvs-psql
+	  exit $?
+	;;
+
+  *)
```
Comment (trevor-jrvs) [inline linux_sql/scripts/psql_docker.sh:45]: indent

File: linux_sql/sql/ddl.sql [hunk 7]
```diff
@@ -0,0 +1,34 @@
+CREATE TABLE PUBLIC.host_info
```
Comment (trevor-jrvs) [inline linux_sql/sql/ddl.sql:1]: include a drop/check

**Review by trevor-jrvs — APPROVED** [review]
fix errors and tidy indentation where indicated

lgtm

File: linux_sql/scripts/host_info.sh (hunk 3 above)
Comment (MoonscriB) [reply linux_sql/scripts/host_info.sh:23]: Not necessarily, you're right!

**Pull request merged**